extensions/wikia/Chat2/ChatBanListSpecialController.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/**
+ * Controller for Wikia's Special:ChatBanList
+ * @author Piotr Molski <moli@wikia-inc.com> for Wikia.com
+ * @author Kuba Karminski <jkarminski@wikia-inc.com> for Wikia.com
+ *
+ */
+class ChatBanListSpecialController extends WikiaSpecialPageController
+{
+
+	function __construct() {
+		parent::__construct( 'ChatBanList', 'chatbanlist' );
+	}
+
+	public function index() {
+		wfProfileIn( __METHOD__ );
+
+		$out = $this->getOutput();
+		$out->addModules( 'ext.Chat2.ChatBanList' );
+		$out->setPageTitle( $this->msg( 'chatbanlist' )->text() );
+		$out->setRobotPolicy( 'noindex,nofollow' );
+		$out->setArticleRelated( false );
```

Comment:
These three lines can be simplified by replacing them with `$this->setHeaders();`